addons/website_slides/static/tests/tours/slides_course_reviews_admin.js

```diff
@@ -0,0 +1,51 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+const messageOfOtherUserSelector =
+    "#chatterRoot:shadow .o-mail-Message:has(.o-mail-Message-textContent:contains('Other user review'))";
+
+/**
+ * This tour test that editing the comment of another user (as admin)
+ * doesn't alter the behavior of the "Edit review" button:
+ * you still edit your review and not the review you've just edited.
+ */
+registry.category("web_tour.tours").add("course_reviews_admin", {
+    url: "/slides",
+    steps: () => [
+        {
+            trigger: "a:contains(Basics of Gardening - Test)",
+            run: "click",
+            expectUnloadPage: true,
+        },
+        {
+            trigger: "a[id=review-tab]",
+            run: "click",
+        },
+        {
+            content: "Display contextual menu of the rating message of another user",
+            trigger: messageOfOtherUserSelector,
+            run: `hover && click ${messageOfOtherUserSelector} [title='Expand']`,
+        },
+        {
+            content: 'Click on "edit" action of the contextual menu of the rating message',
+            trigger: `#chatterRoot:shadow .o-dropdown-item:contains('edit')`,
+            run: "click",
+        },
+        {
+            content: "Update the content of the message",
+            trigger: "#chatterRoot:shadow .o-mail-Message .o-mail-Composer textarea",
+            run: "edit Other user review edited",
+        },
+        {
+            content: 'Click on the "edit review" button to edit the logged user review',
+            trigger: ".o_rating_popup_composer_text:contains(Edit Review)",
+            run: "click",
+        },
+        {
+            content:
+                "Check that the modal displays the message of the logged user and not the one we've just modified",
+            trigger: "div.o_portal_chatter_composer_body textarea:value(Admin review)",
+        },
```

Comment:
As we're here creating a tour (that we may have to remove later) to test this override, I think we should also make sure its intended purpose still works properly as well. Would you mind adding a few steps to edit Admin's review's "email" and witness the popup composer update?